eisvogel.tex

```diff
@@ -846,6 +846,11 @@
 $else$
 {
   \setstretch{1.4}
+  $if(logo)$
```

Comment:
this moves the title logo in front of the author and may break other peoples workflows ...